modules/491-containerized-data-importer/images/artifact/patches/README.md

```diff
@@ -0,0 +1,18 @@
+# Patches
+
+#### `000-bundle-images.patch`
+
+Iternal patch which adds images bundle target with all images to build.
+
+#### `001-deckhouse-registry.patch`
```

Comment:
why the same patch in both new modules?

Reply:
They are different.
One is patching CDI pods, second is patching KubeVirt vms